extensions/banca-d-italia-currency-converter/src/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { Form, ActionPanel, CopyToClipboardAction, showToast, ToastStyle } from "@raycast/api";
+import { useState, useEffect } from "react";
+import { parse as parseCsv } from "papaparse";
+import fetch from "node-fetch";
+
+type CsvRow = {
+  Country: string;
+  Currency: string;
+  "ISO Code": string;
+  "UIC Code": string;
+  Rate: string;
+  "Rate convention": string;
+  "Reference date (CET)": string;
+};
+
+type Rate = {
+  country: string;
+  currency: string;
+  ISOCode: string;
+  rate: number;
+};
+
+const ENDPOINT =
+  "https://tassidicambio.bancaditalia.it/terzevalute-wf-web/rest/v1.0/dailyRates?currencyIsoCode=EUR&lang=en";
```

Comment:
I'm using the same endpoint used by the official web-based conversion tool (https://tassidicambio.bancaditalia.it/terzevalute-wf-ui-web/converter). Unfortunately they use CSV as format... 🤷‍♂️